### PR TITLE
🦋 Update Credit Card Policies form

### PR DESCRIPTION
**Pull request by josemigallas**
[THREESCALE-10077: Update Credit Card Policies](https://issues.redhat.com/browse/THREESCALE-10077)

Before:
Screenshot 2023-08-23 at 09 49 58

After:
Screenshot 2023-08-23 at 11 06 52

### CODE REVIEW

File: features/step_definitions/web_steps.rb [hunk 1]
```diff
@@ -197,6 +197,10 @@ def fill_in
   end
 end
 
+Then "the current page is {}" do |page_name|
+  assert_current_path path_to(page_name)
+end
```
Comment (jlledom) [inline features/step_definitions/web_steps.rb:202]: This looks familiar...

https://github.com/3scale/porta/pull/3526/files#diff-485d7e9c4e7cafba9be795aaae354ff30ac48a9ad9b8a6bf8420e72596bc4995R200-R202

**Review by jlledom — CHANGES_REQUESTED** [review]
This save button needs some margins:

image

Also, the form is not responsive. Should it be?

image

**Comment by josemigallas** [comment]
> This save button needs some margins:
> 
> image
> 
> Also, the form is not responsive. Should it be?
> 
> image

Make sure you've checked out the correct branch @jlledom, those screenshots are the old version

**Comment by jlledom** [comment]
> Make sure you've checked out the correct branch @jlledom, those screenshots are the old version

Yes, I was on the wrong branch. Oops!

**Review by jlledom — APPROVED** [review]

File: features/step_definitions/web_steps.rb (hunk 1 above)
Comment (josemigallas) [reply features/step_definitions/web_steps.rb:202]: 🍒⛏️

**Pull request merged**